### DISCUSSION TITLE
Erreichbarkeit

### DISCUSSION THREAD

**Discussion by 1kartafal4** [McAlex777/Datenschutz#1] [General]
Guten Tag,
kann man Sie per Mail erreichen? Habe einige Fragen zu Ihrem Projekt.
Verschlüsselt mit PGP-Schlüssel bevorzugt.
Gruß

**Comment by McAlex777** [comment]
Hallo,

ich bin hier im Forum erreichbar.

**Comment by githubbapoopa** [comment]
Dann bedanke ich mich statt im Golem Forum auch nochmal hier:

Yup. Hast was auf der Pfanne.

Setzte selbst als Ausgangspunkt auf Win 10 Enterprise (daher trifft nicht alles zu). Für weitere kurze Hilfe/Infos bzgl. der anwendbaren Tool von Dir (in diesem speziellen Anwendungsfall) bin ich dankbar. Bin noch am einlesen...

Deine Zusammenstellung inkl. Aufmachung und Infos sind absolut klasse gemacht.

Vielen Dank!

**Comment by McAlex777** [comment]
Die Scripte sollten grundsätzlich auf Enterprise-Umgebung einsetzbar sein.
Folgende Richtlinien unterscheiden sich unter Windows Enterprise zu PRO, und müssten bei **jedem Update** manuell gesetzt werden.


**Windows10:**
> Computerconfiguration > Administrative Vorlagen > Windows Komponenten > Datensammlung und Vorabversionen > Telemetrie zulassen > AKTIVIERT = 0  

> Benutzerkonfiguration > Administrative Vorlagen > Windows Komponenten > Datensammlung und Vorabversionen > Telemetrie zulassen > AKTIVIERT = 0  


**Windows11:**
> Computerconfiguration > Administrative Vorlagen > Windows Komponenten > Datensammlung und Vorabversionen >  Diagnosedaten zulassen > AKTIVIERT > Diagnosedaten deaktiviert (nicht empfohlen)

> Benutzerkonfiguration > Administrative Vorlagen > Windows Komponenten > Datensammlung und Vorabversionen >  Diagnosedaten zulassen > AKTIVIERT > Diagnosedaten deaktiviert (nicht empfohlen)


**Anmerkungen:**
A)
Wenn die Richtlinien nicht wie o.g. angepasst werden, ist das grundsätzlich kein Problem: 
1. Der Telemetrie-Dienst ist deaktiviert.
2. Der Telemetrie-Dienst ist  via Firewall blockiert.
3. Der lokale Telemetrie-Cache wird via "fwi" beim booten gelöscht. 
4. Erfasst Telemetriestufe 1 keine inhaltlichen Daten, sondern in aller Regel nur Maschinen-Daten.
5. Übertragung von Speicherauszügen beim ProgrammCrash werden gesondert blockiert.

B)
Die Scripte sind für Privat-Umgebungen gebaut. Sobald ein Active-Directory (AD) zur Verfügung steht macht eine zentrale Verwaltung via AD sehrviel mehr Sinn als local GPO. Auch lässt sich der Datenschutz sehrviel effektiver via lokalem WSUS umsetzen.

C)
Wer die Scripte hier einsetzt: bitte gelegentlich vorbeischauen. Es gibt laufend Anpassungen zu neuen Richtlinien, oder kleineren Erweiterungen. 

Ab Windows11 22H2 sollen via Monatsupdates auch Funktionsänderungen ausgerollt werden. Das bedeutet voraussichtlich auch gelegentliche Richtlinien-Anpassungen.

**Comment by githubbapoopa** [comment]
Ok ist angekommen.

Dann sage ich artig Vielen Dank. Werde das hier massiv weiterempfehlen.

Auweia, da wurde ja was losgetreten durch meine kleine Forumsfrage ;)

Klasse, daß Du auch Updates in Aussicht stellst.

**Comment by 1kartafal4** [comment]
Guten Tag,
ich schaue hier immer noch öfters nach.
Sie warten und updaten Ihr Projekt immer noch; vielen Dank dafür.

- Es kostet viel Wissen und Aufwand - Sie möchten aber keine Spenden annehmen?
- Finde es ein Unding dass Ihr Projekt nahezu unbekannt ist. Sie möchten dass es so bleibt?

Updaten und warten Sie Ihre IP-Liste (Blockieren) mit Ein-/Ausgängen zu Microsofts/Azure/Office etc. Servern?

Gruß

**Comment by 1kartafal4** [comment]
Sind Sie noch unter der E-Mail Adresse "mc*****@gmail.com" erreichbar, bzw. möchten erreichbar sein?
Da antwortet niemand.
Gruß
